react/components/Tooltip/TooltipPopup.tsx

```diff
@@ -100,6 +100,7 @@ const TooltipPopup: FC<PropTypes.InferProps<typeof propTypes>> = ({
           ...positionStyle,
           zIndex: zIndex.tooltip,
           transition: `opacity ${duration}ms ${timmingFn} ${delay}ms`,
+          wordBreak: 'break-all',
```

Comment:
Actually, it could be a prop. This is because the `break-all` property really breaks the word itself and, in many cases, we don't want that. It's more often to want to break in whitespace, so... Could you add a prop with it? It could be a `wordBreak` prop or even a `popupStyle` prop

Reply:
@emersonlaurentino super large words didn't need to be handled before. I also think that changing the `max-width` could be better, but I didn't make these experiments and surely @lucasaarcoverde should have done it.

Reply:
`wordBreak` is the best solution, since we want the opinioned components.

Reply:
I'd say static components but yeah... It can be `wordBreak` 😄

Reply:
@estacioneto i did this test of change `max-width` and it works, but i talked to @oldufre and the best solution is using `wordBreak`. She did some research and found that is how other style guides deal with it

Reply:
Great job! As I thought you guys would have done. But anyway can you add it as a prop instead?

Reply:
I will change the `wordBreak` to be a prop :smile: